docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -876,6 +876,83 @@ To retrieve the information about the data files of the Iceberg table ``test_tab
 
 
 
+The output of the query has the following columns:
+
+.. list-table:: Files columns
+  :widths: 25, 30, 45
+  :header-rows: 1
+
+  * - Name
+    - Type
+    - Description
+  * - ``content``
+    - ``integer``
+    - Type of content stored in the file.
+      The supported content types in Iceberg are:
+
+      * ``DATA(0)``
+      * ``POSITION_DELETES(1)``
+      * ``EQUALITY_DELETES(2)``
+  * - ``file_path``
+    - ``varchar``
+    - The data file location
+  * - ``file_format``
+    - ``varchar``
+    - The format of the data file
+  * - ``record_count``
+    - ``bigint``
+    - The number of entries contained in the data file
+  * - ``file_size_in_bytes``
+    - ``bigint``
+    - The data file size
+  * - ``column_sizes``
+    - ``map(integer, bigint)``
+    - Mapping between the Iceberg column ID and its corresponding size in the file
+  * - ``value_counts``
+    - ``map(integer, bigint)``
+    - Mapping between the Iceberg column ID and its corresponding count of entries in the file
+  * - ``null_value_counts``
+    - ``map(integer, bigint)``
+    - Mapping between the Iceberg column ID and its corresponding count of ``NULL`` values in the file
+  * - ``nan_value_counts``
+    - ``map(integer, bigint)``
+    - Mapping between the Iceberg column ID and its corresponding count of non numerical values in the file
+  * - ``lower_bounds``
+    - ``map(integer, bigint)``
+    - Mapping between the Iceberg column ID and its corresponding lower bound in the file
+  * - ``upper_bounds``
+    - ``map(integer, bigint)``
+    - Mapping between the Iceberg column ID and its corresponding upper bound in the file
+  * - ``key_metadata``
+    - ``varbinary``
+    - Metadata about the encryption key used to encrypt this file, if applicable
+  * - ``split_offsets``
+    - ``array(bigint)``
+    - List of recommended split locations.
+  * - ``equality_ids``
+    - ``array(integer)``
+    - The set of field IDs used for equality comparison in equality delete files
+
+``$all_files`` table
+^^^^^^^^^^^^^^^^^^^^
+
+The ``$all_files`` table exposes the valid data files of the Iceberg table.
+A valid data file is one that is readable from any snapshot currently tracked by the table.
+
+To retrieve the information about the data files from all the snapshots of the Iceberg table ``test_table`` use the following query::
+
+    SELECT * FROM "test_table$all_files"
+
+The output of the query has the columns, which is similar to the ``$files`` metadata table.
+
+.. code-block:: text
+
+     content  | file_path                                                                                                                     | record_count    | file_format   | file_size_in_bytes   |  column_sizes        |  value_counts     |  null_value_counts | nan_value_counts  | lower_bounds                |  upper_bounds               |  key_metadata  | split_offsets  |  equality_ids
```

Comment:
Please replace the order > `record_count` & `file_format`.

Reply:
updated

Reply:
> @osscm CI is still failing.

@ebyhr  trying to fix.

Reply:
> @osscm CI is still failing.

yeah, changed the code as well (as Iceberg 0.14 has introduced the `DeleteFiles` table as well, so we have to DataFiles to keep the existing FilesTable behavior), which helped to resolve the 4 test failures

Verified in local as well, and in the `test(plugin/trino-iceberg)` logs that all of these 4 tests ran successfully.
Is there an easier way to check the test report from the builds?
 
```
TestIcebergV2.testOptimizingPartitionsOfV2TableWithGlobalEqualityDeleteFile:280->getActiveFiles:529->AbstractTestQueryFramework.computeActual:212->AbstractTestQueryFramework.computeActual:217 » QueryFailed
TestIcebergV2.testOptimizingV2TableDoesntRemoveEqualityDeletesWhenOnlyPartOfTheTableIsOptimized:222->getActiveFiles:529->AbstractTestQueryFramework.computeActual:212->AbstractTestQueryFramework.computeActual:217 » QueryFailed
TestIcebergV2.testOptimizingV2TableRemovesEqualityDeletesWhenWholeTableIsScanned:197->getActiveFiles:529->AbstractTestQueryFramework.computeActual:212->AbstractTestQueryFramework.computeActual:217 » QueryFailed
TestIcebergV2.testOptimizingV2TableWithEmptyPartitionSpec:261->getActiveFiles:529->AbstractTestQueryFramework.computeActual:212->AbstractTestQueryFramework.computeActual:217 » QueryFailed
```